Add note to install mailcap package for app functionality (#3954)
mattermost/docs: source/install/install-rhel-7-server.rst

  ``sudo yum upgrade``

Now that the system is up to date, you can start installing the components that make up a Mattermost system.

.. note:: 

>   Please make sure the ``mailcap`` package is intalled as the ``mime.types`` file it installs is needed for the mobile app to work correctly.

Reviewer: ```suggestion
  Ensure that the ``mailcap`` package is installed as it includes the ``mime.types`` file which is needed for the Mobile App to work correctly.
```
